backend/helper/S3Helper.js

```diff
@@ -216,6 +218,37 @@ module.exports.getAllObjects = async (s3, bucketName) => {
 
 };
 
+module.exports.getAllObjectsByDirectory = async (s3, bucketName, prefix) => {
+
+    const data = await s3.send(new ListObjectsCommand({
+        Bucket: bucketName,
+        Delimiter: "/",
+        Prefix: prefix,
+    }));
+
+    console.log(data);
+
+    let folders = data.CommonPrefixes ? data.CommonPrefixes : [];
+
+    let objects = data.Contents ? data.Contents : [];
+    let nextMarker = data.NextMarker;
+
+    while (nextMarker){
```

Comment:
امکانش نیست به این شکل نوشته بشه؟ اینجوری دو خط کم میشه و خوانایی کد هم بیشتر میشه یک متغیر هم حذف میشه. :)

```suggestion
    while (data.NextMarker){
```

Reply:
آره بهتره.